cg/cli/upload/vogue.py

```diff
@@ -1,22 +1,35 @@
 """Base command for trending"""
 
+import datetime as dt
 import logging
 from pathlib import Path
 from typing import Any, List, Optional, Tuple
 
 import click
+
 from cg.apps.housekeeper.hk import HousekeeperAPI
 from cg.constants import Pipeline
 from cg.exc import AnalysisUploadError
 from cg.meta.upload.vogue import UploadVogueAPI
 from cg.meta.workflow.mip_dna import MipDNAAnalysisAPI
 from cg.models.cg_config import CGConfig
 from cg.store import Store, models
+from cg.store.api.update import update_analysis_uploaded_to_vogue_date
 from housekeeper.store import models as hk_models
 
 LOG = logging.getLogger(__name__)
 
-VOGUE_VALID_BIOINFO = [str(Pipeline.MIP_DNA), str(Pipeline.BALSAMIC)]
+VOGUE_VALID_BIOINFO = [str(Pipeline.BALSAMIC)]
```

Comment:
why remove mip from valid pipelines?

Reply:
That's by request of @hassanfa

Reply:
Yes. Mip is running a hybrid qc analysis on wgs cases that it is hard to handle right now. But we need data in vogue to be able to continue planning the frontend. When that's done we'll allow exceptions such as mip's qc output.

Reply:
Could this not be handled by stratifying the input data on the application tag or filtering the WGS data for certain metrics like picards HS when plotting? It seems pretty drastic to turn off an entire workflow in production.

Reply:
Vogue for cancer and rare disease bioinfo is not in production yet. So nothing will suffer.

Well first we need high amount of data in Vogue to be able to properly plan the frontend. I mean, I can completely turn off WGS in Vogue irregardless of workflow, but I want to first see what we want and then re-upload.

That's why Barry and I decided to make sure we can upload to Vogue based on date limit. This way, after we have fixed Vogue that can properly accept exceptions with a date range, then we will upload without exception.

WGS is always been very high quality data. TGA and WES with BALSAMIC needs to be uploaded ASAP to help lab with their troubleshooting.

There is a issue to also upload to Vogue with workflow specific data: https://github.com/Clinical-Genomics/cg/issues/926